### PR TITLE
[tensorflow(-cc)] no absolute paths

### PR DESCRIPTION
**Pull request by autoantwort**
For #20469

### CODE REVIEW

File: ports/tensorflow-cc/portfile.cmake [hunk 1]
```diff
@@ -6,7 +6,7 @@ endif()
 
 set(TF_LIB_SUFFIX "_cc")
 set(TF_PORT_SUFFIX "-cc")
-set(TF_INCLUDE_DIRS "${CURRENT_INSTALLED_DIR}/include/tensorflow-external ${CURRENT_INSTALLED_DIR}/include/tensorflow-external/src")
+set(TF_INCLUDE_DIRS "\${TENSORFLOW_INSTALL_PREFIX}/include/tensorflow-external \${TENSORFLOW_INSTALL_PREFIX}/include/tensorflow-external/src")
```
Comment (JackBoosY) [inline ports/tensorflow-cc/portfile.cmake:9]: I'm not sure whether it's correct.
`TENSORFLOW_INSTALL_PREFIX` is used in https://github.com/microsoft/vcpkg/blob/eff88baadcf16a7c441bf42be7eafb1df7d37d4e/ports/tensorflow-common/tensorflow-common.cmake#L424
but overwritten by https://github.com/microsoft/vcpkg/blob/030cfaa24de9ea1bbf0a4d9c615ce7312ba77af1/ports/tensorflow-common/tensorflow-config-shared.cmake.in#L14
or
https://github.com/microsoft/vcpkg/blob/030cfaa24de9ea1bbf0a4d9c615ce7312ba77af1/ports/tensorflow-common/tensorflow-config-static.cmake.in#L9

Comment (JackBoosY) [reply ports/tensorflow-cc/portfile.cmake:9]: cc @jgehw for review this part.

Comment (autoantwort) [reply ports/tensorflow-cc/portfile.cmake:9]: It is the same as `_IMPORT_PREFIX` in other ports.

Comment (JackBoosY) [reply ports/tensorflow-cc/portfile.cmake:9]: I would like @jgehw to confirm this.

**Review by JackBoosY — APPROVED** [review]

**Comment by BillyONeal** [comment]
Thanks :)

**Pull request merged**